fars/booking/gcal.py

```diff
@@ -0,0 +1,215 @@
+from getenv import env
+from fars.settings import TIME_ZONE
+from django.utils import timezone
+from googleapiclient.discovery import build
+from google.oauth2.service_account import Credentials
+from googleapiclient.errors import HttpError
+from threading import Thread
+import logging, json
+
+logger = logging.getLogger(__name__)
+
+gcal_service = build(
+    'calendar',
+    'v3',
+    credentials=Credentials.from_service_account_file(
+        env('GOOGLE_SERVICE_ACCOUNT_FILE'),
+        scopes=['https://www.googleapis.com/auth/calendar'],
+    )
+)
```

Comment:
My python is a bit rusty... but when exactly is this code executed? on import? If so, then it might get a bit unwieldy to use, I would not expect exceptions to be thrown on import. Preferably it should happen in some clear place, constructor or, preferably some factory method.